.travis.yml

```diff
@@ -23,41 +23,39 @@ env:
         - PIP_DEPENDENCIES=''
         - CRDS_SERVER_URL='https://jwst-crds.stsci.edu'
         - CRDS_PATH='/tmp/crds_cache'
-        - NUMPY_VERSION=1.12
+        - PYTHON_VERSION=3.6
```

Comment:
@hbushouse , I made Python 3.6 the default Python version unless stated otherwise here.